mkosi.build

```diff
@@ -20,11 +20,15 @@
 # This is a build script for OS image generation using mkosi (https://github.com/systemd/mkosi).
 # Simply invoke "mkosi" in the project directory to build an OS image.
 
+# If mkosi.builddir/ exists mkosi will set $BUILDDIR to it, let's then use it
+# as out-of-tree build dir. Otherwise, let's make up our own builddir.
+[ x"$BUILDDIR" = x ] && BUILDDIR=build
```

Comment:
hmm, this didn't terminate the script... Not sure why though, but I tested the script both with mkosi.builddir around and without and it worked both times and did the right thing.